themes/default/content/blog/announcing-kubernetes-helm-release-resource-preview/index.md

```diff
@@ -0,0 +1,305 @@
+---
+title: Announcing Preview for Helm Release Resource for Kubernetes
+h1: "Announcing Preview for Helm Release Resource for Kubernetes"
+date: "2021-09-07"
+meta_desc: "Today we are announcing preview of a new Helm Release resource for Kubernetes which adds support for Helm natively on Pulumi."
+meta_image: "helm-release.gif"
+authors: ["vivek-lakshmanan"]
+tags: ["Kubernetes", "helm"]
+---
+
+Kubernetes has been a significant focus of Pulumi since its very beginnings. Pulumi added support for installing [Helm charts](https://helm.sh/) way back in [2018](https://www.pulumi.com/blog/using-helm-and-pulumi-to-define-cloud-native-infrastructure-as-code/) and it has seen significant adoption by users since. However, Pulumi's current Chart integration lacks support for some increasingly common advanced features in Helm charts, e.g.:
+
+1. Support for [Helm lifecycle hooks](https://helm.sh/docs/topics/charts_hooks/)
+2. Handling sub-charts and [dependencies](https://helm.sh/docs/topics/charts/#chart-dependencies)
+
+These and other limitations have resulted in Helm support becoming a common painpoint for our Kubernetes users.
```

Comment:
```suggestion
As Helm continues to evolve, Pulumi users using the `Chart` resource have often had to get very creative in order to get the desired functionality
```

Tried to put a more positive spin on this

Reply:
Updated. Could you take a look to make sure you agree?